src/metaArray.cxx

```diff
@@ -612,7 +612,7 @@ MetaArray::Read(const char * _headerName, bool _readElements, void * _elementDat
 
 
 bool
-MetaArray::CanReadStream(std::ifstream * _stream) const
+MetaArray::CanReadStream(METAIO_STREAM::ifstream * _stream) const
```

Comment:
Does this even need to be ifstream? Isn't istream sufficient?
kwsys::ifstream inherits from std::istream and kwsys::ofstream inherits from std::ostream. I suppose there are many places in the code where using istream/ostream parameters would be preferable so that METAIO_STREAM does not need to be referenced at all.

Reply:
Really good point.  Do you think those changes should be explored in this or a separate pull request?   My suggestion is separate - we'll need the ITK folks to weigh-in on it.

Reply:
A separate pull request is probably better.